classes/Http/Action/Profile/PasswordAction.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2017 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Http\Action\Profile;
+
+use Symfony\Component\HttpFoundation;
+use Twig_Environment;
+
+final class PasswordAction
+{
+    /**
+     * @var Twig_Environment
+     */
+    private $twig;
+
+    public function __construct(Twig_Environment $twig)
+    {
+        $this->twig = $twig;
+    }
+
+    public function __invoke(): HttpFoundation\Response
+    {
+        $content = $this->twig->render('user/change_password.twig');
```

Comment:
Since we already have a bunch of actions which only render templates, we could probably consider extracting a `StaticPageAction`, and inject the template into it:

```php
<?php

declare(strict_types=1);

/**
 * Copyright (c) 2013-2017 OpenCFP
 *
 * For the full copyright and license information, please view
 * the LICENSE file that was distributed with this source code.
 *
 * @see https://github.com/opencfp/opencfp
 */

namespace OpenCFP\Http\Action;

use Symfony\Component\HttpFoundation;
use Twig_Environment;

final class StaticPageAction
{
    /**
     * @var Twig_Environment
     */
    private $twig;
    
    /**
     * @var string
     */
    private $template;

    public function __construct(Twig_Environment $twig, string $template)
    {
        $this->twig = $twig;
        $this->template = $template;
    }

    public function __invoke(): HttpFoundation\Response
    {
        $content = $this->twig->render($this->template);

        return new HttpFoundation\Response($content);
    }
}
```

Probably makes sense to wait with this until #895 is sorted out.

Reply:
@localheinz Symfony has a `TemplateController` that does exactly that.

https://github.com/symfony/symfony/blob/3.4/src/Symfony/Bundle/FrameworkBundle/Controller/TemplateController.php

Reply:
Once #895 is merged, I can prepare a PR that switches the static actions to that controller. Until then, it would be hard to keep that PR in sync with the routing changes.